src/witan/models/dem/ccm/schemas.clj

```diff
@@ -41,7 +41,8 @@
                            [:death-rate s/Num] [:popn s/Num] [:deaths s/Num]]))
 
 (def NetMigrationSchema
-  (make-ordered-ds-schema [[:gss-code s/Str] [:sex s/Str] [:age s/Int] [:net-mig s/Num]]))
+  (make-ordered-ds-schema [[:gss-code s/Str] [:sex s/Str] [:age s/Int] [:domin s/Num]
+                           [:intin s/Num] [:domout s/Num] [:intout s/Num] [:net-mig s/Num]]))
```

Comment:
this does not _need_ to contain other migration results other than the net mig, as net mig is what is needed in next step, so wondering why its here?